evap/context_processors.py

```diff
@@ -0,0 +1,10 @@
+from django.conf import settings
+
+def feedback_email(request):
+    return {'FEEDBACK_EMAIL': settings.FEEDBACK_EMAIL}
+
+def legal_notice(request):
+    return {'LEGAL_NOTICE_LINK': settings.LEGAL_NOTICE_LINK}
+
+def tracker_url(request):
+    return {'TRACKER_URL': settings.TRACKER_URL}
```

Comment:
what i find weird is that we have this settings available in all templates. when doing template tags, we would only have them available in base.html and impressum.html. in my opinion that's better, but i'm not completely sure.